src/backend/gl/src/tex.rs

```diff
@@ -208,7 +207,10 @@ fn format_to_glfull(format: NewFormat) -> Result<GLenum, ()> {
             C::Float => gl::RGBA32F,
             _ => return Err(()),
         },
-        S::B8_G8_R8_A8 => return Err(()), // TODO
+        S::B8_G8_R8_A8 => match cty {
+            C::Unorm => gl::RGBA8,
```

Comment:
So, if I understand correctly, the texture data is still stored as `RGBA8` but what you want to specify under `BGRA` format is that the data you provide is swizzled?